### PR TITLE
[KYUUBI # 5085] Update config section of deploy on kubernetes

### PR DESCRIPTION
**Pull request by mans2singh**
### _Why are the changes needed?_
Updated [kyuubi on kubernetes config section](https://kyuubi.readthedocs.io/en/master/deployment/kyuubi_on_kubernetes.html#config) to state <code> Kyuubi **does** not recommend using this way on Kubernetes</code>

### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [ ] [Run test](https://kyuubi.readthedocs.io/en/master/contributing/code/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

File: docs/deployment/kyuubi_on_kubernetes.md [hunk 1]
```diff
@@ -90,7 +90,7 @@ See more related details in [Using RBAC Authorization](https://kubernetes.io/doc
 
 ## Config
 
-You can configure Kyuubi the old-fashioned way by placing kyuubi-default.conf inside the image. Kyuubi do not recommend using this way on Kubernetes.
+You can configure Kyuubi the old-fashioned way by placing kyuubi-default.conf inside the image. Kyuubi does not recommend using this way on Kubernetes.
```
Comment (pan3793) [inline docs/deployment/kyuubi_on_kubernetes.md:93]: There is another issue in this statement

```suggestion
You can configure Kyuubi the old-fashioned way by placing `kyuubi-defaults.conf` inside the image. Kyuubi does not recommend using this way on Kubernetes.
```

Comment (mans2singh) [reply docs/deployment/kyuubi_on_kubernetes.md]: Thanks @pan3793 - I've updated the docs with your correction.

**Review by pan3793 — APPROVED** [review]

**Comment by pan3793** [comment]
Thanks, merged to master/1.7

**Comment by mans2singh** [comment]
Thanks @pan3793

**Pull request closed without merging**